e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,181 @@
+    switch (userLockout) {
+      case 'DISABLE':
+        expect(status).toEqual('Active');
+        break;
+      case 'ENABLE':
+        expect(status).toEqual('Disabled');
+        break;
+    }
+
+    //get the name column index separately since it is a frozen column
+    const nameColIndex = await adminTable.getNameColindex();
+
+    //click on the name link to navigate to the admin-user-profile page
+    const userProfileInfo = await userAdminPage.clickNameLink(1, nameColIndex);
+
+    //navigate to admin-user-profile page
+    await userProfileInfo.waitForLoad();
+
+    //verify that the save button is disabled
+    expect(await userProfileInfo.getSaveButton().isCursorNotAllowed()).toBe(true);
+
+    // get the Account Access toggle status (true or false) and the text (enabled or disabled)
+    const accountAccessToggle = userProfileInfo.getAccountAccessToggle();
+    const accountAccessStatus = await userProfileInfo.getAccountAccessText();
+
+    //switch statement to validate that the Account Access text and toggle reflects the user lockout on user-admin-table.
+    switch (userLockout) {
+      case 'DISABLE':
+        expect(accountAccessStatus).toEqual('Enabled');
+        expect(await accountAccessToggle.isChecked()).toBe(true);
+        break;
+      case 'ENABLE':
+        expect(accountAccessStatus).toEqual('Disabled');
+        expect(await accountAccessToggle.isChecked()).toBe(false);
+        break;
+    }
+
+    //verify that userfulName, username and FreeCreditsUsed input fields are disabled
+    expect(await userProfileInfo.getNameInput().isCursorNotAllowed()).toBe(true);
+    expect(await userProfileInfo.getUsernameInput().isCursorNotAllowed()).toBe(true);
+    expect(await userProfileInfo.getFreeCreditsUsedInput().isCursorNotAllowed()).toBe(true);
+
+    // get the username field placeholder to verify the email on user-admin-table page and admin-user-profile page matches
+    const userNamePlaceHolder = await userProfileInfo.getUserNamePlaceHolder();
+    expect(userNameEmail).toEqual(userNamePlaceHolder);
+
+    //verify the credit limit value matches the max value in the free credits used field
+    const freeCreditLimit = await userProfileInfo.getFreeCreditsLimitValue();
+    const freeCreditMaxValue = await userProfileInfo.getFreeCreditMaxValue();
+    expect(freeCreditLimit).toEqual(freeCreditMaxValue);
+
+    //get the bypass access status of all modules on admin-user-profile page
+    const twoFAToggleStatus2 = await userProfileInfo.getEachBypassStatus('twoFactorAuthBypassToggle');
+    const trainingToggle2 = await userProfileInfo.getEachBypassStatus('complianceTrainingBypassToggle');
+    const eraCommToggleStatus2 = await userProfileInfo.getEachBypassStatus('eraCommonsBypassToggle');
+    const dUCCToggleStatus2 = await userProfileInfo.getEachBypassStatus('dataUseAgreementBypassToggle');
+
+    //verify all the modules status on user-admin-table page and admin-user-profile page match
```

Comment:
these steps will verify that the modules-bypass status on user-admin page & user-profile page matches.